module/zfs/zio.c

```diff
@@ -5023,6 +5023,18 @@ zbookmark_subtree_completed(const dnode_phys_t *dnp,
 	    last_block) <= 0);
 }
 
+boolean_t
+zbookmark_subtree_tbd(const dnode_phys_t *dnp,
```

Comment:
I'm probably overlooking that obvious, but what does `_tbd` stand for here?  I'd also find it helpful if you added a comment similar in spirit to the one above `zbookmark_subtree_completed()` to explain what's being checked here.

Reply:
"To Be Done", as partial opposite to "complete" above.  As alternative I was thinking of "Greater or Equal", but any better ideas are welcome.

Reply:
Added comment and changed ASSERT() to ASSERT0().